app/uk/gov/hmrc/customerprofile/services/ApplePassService.scala

```diff
@@ -47,10 +47,11 @@ class ApplePassService @Inject()(citizenDetailsConnector: CitizenDetailsConnecto
       for {
         nino <- getNino()
         citizenDetails <- citizenDetailsConnector.personDetails(nino.getOrElse(throw new NinoNotFoundOnAccount("")))
-        getApplePassUUID <- createApplePassConnector.createApplePass(nino.getOrElse(throw new NinoNotFoundOnAccount("")), citizenDetails.person.completeName)
-        getApplePassGenerator <- createApplePassConnector.getPass(getApplePassUUID.uuid)
-        applePass: RetrieveApplePass = RetrieveApplePass(getApplePassGenerator.applePass)
-      } yield applePass
+        createApplePass <- createApplePassConnector.createApplePass(nino.getOrElse(throw new NinoNotFoundOnAccount("")),
```

Comment:
Can we improve the NotFoundException logging as we discussed in ErrorHandling.scala please.